lib/DDGC/Schema/Result/User/Blog.pm

```diff
@@ -0,0 +1,186 @@
+package DDGC::Schema::Result::User::Blog;
+
+# ABSTRACT: Result class of blog posts
+
+use Moo;
+extends 'DDGC::Schema::Result';
+use DBIx::Class::Candy;
+use DateTime::Format::RSS;
+use DDGC::Util::Markup;
+
+table 'user_blog';
+
+column id => {
+    data_type         => 'bigint',
+    is_auto_increment => 1,
+};
+primary_key 'id';
+
+column users_id => {
+    data_type   => 'bigint',
+    is_nullable => 0,
+};
+
+column translation_of_id => {
+    data_type   => 'bigint',
+    is_nullable => 1,
+};
+
+column title => {
+    data_type   => 'text',
+    is_nullable => 0,
+};
+
+column uri => {
+    data_type   => 'text',
+    is_nullable => 0,
+};
+
+column teaser => {
+    data_type   => 'text',
+    is_nullable => 0,
+};
+
+sub html_teaser {
+    my ($self) = @_;
+    my $markup = DDGC::Util::Markup->new;
+    return $markup->html( $self->teaser ) if $self->raw_html;
+    return $markup->bbcode( $self->teaser );
+}
+
+column content => {
+    data_type   => 'text',
+    is_nullable => 0,
+};
+
+sub html {
+    my ($self) = @_;
+    my $markup = DDGC::Util::Markup->new;
+    return $markup->html( $self->content ) if $self->raw_html;
+    return $markup->bbcode( $self->content );
+}
+
+column topics => {
+    data_type        => 'text',
+    is_nullable      => 1,
+    serializer_class => 'JSON',
+};
+
+# TODO: Deprecate this
+column company_blog => {
+    data_type     => 'int',
+    is_nullable   => 0,
+    default_value => 0,
+};
+
+column raw_html => {
+    data_type     => 'int',
+    is_nullable   => 0,
+    default_value => 0,
+};
+
+column live => {
+    data_type          => 'int',
+    is_nullable        => 0,
+    default_value      => 0,
+    keep_storage_value => 1,
+};
+
+column seen_live => {
+    data_type     => 'int',
+    is_nullable   => 0,
+    default_value => 0,
+};
+
+column language_id => {
+    data_type   => 'bigint',
+    is_nullable => 1,
+};
+
+column data => {
+    data_type        => 'text',
+    is_nullable      => 1,
+    serializer_class => 'JSON',
+};
+
+column fixed_date => {
+    data_type   => 'timestamp with time zone',
+    is_nullable => 1,
+};
+
+column created => {
+    data_type     => 'timestamp with time zone',
+    set_on_create => 1,
+};
+
+column updated => {
+    data_type     => 'timestamp with time zone',
+    set_on_create => 1,
+    set_on_update => 1,
+};
+
+belongs_to 'user', 'DDGC::Schema::Result::User', 'users_id';
+
+after insert => sub {
```

Comment:
This would trigger event generation for notifications. Genuinely, I've read the code many times and have no idea what `seen_live` is really supposed to be for.

Shim for existing notification system compat coming (I think this is easy..), `seen_live`probably needs to go away.